elastic/apm-server
# Move setting destination.ip back into apm-server

## Motivation/summary

Revert the addition of the `apm_convert_destination_address` ingest pipeline, and set the `destination.ip` field during
model transformation. We can move it back to ingest node once we can rely on Elasticsearch supporting the `ip` type in the `convert` processor. While we're at it, move Destination out of Span, and up to APMEvent.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~
~- [ ] Documentation has been updated~

## How to test these changes

Send some spans which result in `destination.address` being set -- one with an IP value, one with a hostname. Both should be indexed, and `destination.address` should also be copied to `destination.ip` in the IP value case.

## Related issues

Fixes #5898
